.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: go
 go:
-  - 1.8
-  - tip
+  - 1.13
 install:
-  - go get github.com/constabulary/gb/...
+  - go mod download
```

Comment:
Interesting, I've not seen this before. Why use this when `go build` will download all dependencies though?